src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_RootScreen/components/Announcement.tsx

```diff
@@ -45,91 +45,95 @@ export const Announcement = ({announcement, style}: Props) => {
-                  height={50}
-                  width={50}
-                  source={{uri: announcement.summaryImage}}
-                />
-              </View>
-            )}
-            <View style={styles.textContainer}>
-              <ThemeText type="body__primary--bold">
-                {getTextForLanguage(
-                  announcement.summaryTitle ?? announcement.fullTitle,
-                  language,
+      <GenericSectionItem style={styles.sectionItem}>
+        <View style={styles.content}>
+          {announcement.summaryImage && (
+            <View style={styles.imageContainer}>
+              <Image
+                height={50}
+                width={50}
+                source={{uri: announcement.summaryImage}}
+              />
+            </View>
+          )}
+          <View style={styles.textContainer}>
+            <View style={styles.summaryTitle}>
+              <ThemeText type="body__primary--bold">{summaryTitle}</ThemeText>
+              <PressableOpacity
+                style={styles.close}
+                role="button"
+                hitSlop={insets.all(theme.spacings.medium)}
+                accessibilityHint={t(
+                  DashboardTexts.announcemens.announcement.closeA11yHint,
                 )}
-              </ThemeText>
-              <ThemeText style={styles.summary}>
-                {getTextForLanguage(announcement.summary, language)}
-              </ThemeText>
+                onPress={() => handleDismiss()}
+                testID="closeAnnouncement"
+              >
+                <ThemeIcon svg={Close} />
+              </PressableOpacity>
             </View>
+            <ThemeText style={styles.summary}>
+              {getTextForLanguage(announcement.summary, language)}
+            </ThemeText>
           </View>
-          <PressableOpacity
-            style={styles.close}
-            role="button"
-            hitSlop={insets.all(theme.spacings.medium)}
-            accessibilityHint={t(
-              DashboardTexts.announcemens.announcement.closeA11yHint,
-            )}
-            onPress={() => handleDismiss()}
-            testID="closeAnnouncement"
-          >
-            <ThemeIcon svg={Close} />
-          </PressableOpacity>
         </View>
       </GenericSectionItem>
-      {announcement.actionButton?.actionType &&
-        announcement.actionButton?.label && (
-          <LinkSectionItem
-            text={
-              getTextForLanguage(announcement.actionButton.label, language) ??
-              ''
-            }
-            icon={
-              announcement.actionButton?.actionType === 'external'
-                ? 'external-link'
-                : 'arrow-right'
-            }
-            accessibility={{
-              accessibilityHint: t(
-                DashboardTexts.announcemens.buttonAction[
-                  announcement.actionButton.actionType
-                ],
+      {announcement.actionButton?.actionType && (
```

Comment:
With the fallback to `bottom_sheet` in the converter, the `actionType` doesn't need to be optional in the typing now?

Reply:
Yes, I suppose that's true 👍

Reply:
Fixed in https://github.com/AtB-AS/mittatb-app/pull/4148/commits/f24e8f6902468c0ca282d03c8d8573277df608eb